src/startup/mapping.jsx

```diff
@@ -0,0 +1,12 @@
+import { routes } from 'startup/routes'
+import { Route } from 'react-router-dom'
+import React from 'react'
+
+export const Mapping = () => <>
+  {
+    routes.map((route, index) => <Route path={route.Path} key={index}>
```

Comment:
But we can make this `map` function even better. The only restriction towards what value you use as a `key` prop is its uniqueness and since we already know that each `path` field within our `routes` array will be unique, we can use that instead of that index. That means we can remove the `index` entirely.